src/screens/Lessons/screens/Lesson/components/LessonData/index.js

```diff
@@ -79,21 +65,26 @@ export default ({instructor, lesson}) => {
             key={index}
             className={`${index < (size(items) - 1) ? 'bb' : ''} pb3 mb3 b--gray`}
           >
-            <Heading level='4'>
+            <h4 className='mt0 bold f4 mb2 white-70'>
               {item.title}
-            </Heading>
+            </h4>
             <div>
               {item.children}
             </div>
           </div>
         ))}
       </div>
 
-      {isInstructorsOwnLesson 
+      {lesson.edit_lesson_http_url
```

Comment:
still using a temp bridge to egghead.io proper, but using the hypermedia instead of checking ownership